tests/test_pulp_util.py

```diff
@@ -197,3 +198,69 @@ def test_get_pulp_instance():
     _, workflow = prepare(testfile)
     handler = PulpHandler(workflow, pulp_registry_name, log)
     assert handler.get_pulp_instance() == pulp_registry_name
+
+
+@pytest.mark.skipif(dockpulp is None,
+                    reason='dockpulp module not available')
+@pytest.mark.parametrize(("auto_publish"), [
+    (True),
+    (False),
+])
+@pytest.mark.parametrize(("fail"), [
+    (True),
+    (False)
+])
+def test_ensure_repos(auto_publish, fail):
+    dist_data = [
+          {
+            "repo_id": "redhat-myproject-hello-world",
+            "auto_publish": auto_publish,
+          }
+        ]
+    mock_get_data = [
+      {
+        "id": "redhat-myproject-hello-world",
+        "scratchpad": {
```

Comment:
Is `scratchpad`, `tags`, `display_name` and `description` relevant to the test? It seems to clutter the test case.